core/src/main/java/io/servicecomb/core/SystemBootListener.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicecomb.core;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SystemBootListener implements BootListener {
+
+  private static boolean ready = false;
```

Comment:
I see many guidelines saying better to give a default value implicitly. e.g. some eclipse compiler and coverity tools